src/plugins/brightcovePlugin.js

```diff
@@ -136,36 +137,48 @@ export default function createBrightcovePlugin(options = { concept: false }) {
       source: t(locale, 'source'),
     };
 
-    const id = brightcove.reference_id
-      ? brightcove.reference_id
-      : brightcove.id;
-    const figureId = `figure-${id}`;
+    const figureId = `figure-${brightcove.id}`;
 
     return render(
       <Figure
         id={figureId}
         type={options.concept ? 'full-column' : 'full'}
         resizeIframe>
-        <iframe
-          title={`Video: ${brightcove.name}`}
-          aria-label={`Video: ${brightcove.name}`}
-          frameBorder="0"
-          {...getIframeProps(data, brightcove.sources)}
-          allowFullScreen
-        />
+        <div className="brightcove-video">
+          <iframe
+            className="original"
+            title={`Video: ${brightcove.name}`}
+            aria-label={`Video: ${brightcove.name}`}
+            frameBorder="0"
+            {...getIframeProps(data, brightcove.sources)}
+            allowFullScreen
+          />
+          <iframe
+            className="synstolket hidden"
```

Comment:
Blir denne lazyloadet eller vil sider med videoer nå trenge å laste inn to iframes med videoer i?

Reply:
Den blir nok ikke lazy nei